docs/gitbook/architecture/topology/patterns.adoc

```diff
@@ -0,0 +1,148 @@
+= Topology Patterns
+
+The blockchain topology patterns used by Interbit are:
+
+* <<_atom>>
+* <<_molecules>>
+** <<_hub_and_spoke>>
+** <<_hub_and_connected_spoke>>
+** <<_control_by_proxy>>
+
+The <<_combinations>> section describes more advanced topologies.
+
+
+== Atom
+
+An _atom_ is a single Interbit blockchain with a single
+smart contract running on it.
+
+With Interbit different smart contracts
+are different atoms. As with atoms, Interbit blockchains can come in a
+variety of types (within a solution) based on what smart contract runs
+on that blockchain. It is an important part of architecting a solution
+to identify how many distinct smart contracts there are. This will
```

Comment:
Suggestion:  "Identifying how many distinct smart contracts are needed is an important part of architecting a solution."